lib/bloc/account/fiat_conversion.dart

```diff
@@ -32,6 +32,10 @@ class FiatConversion {
     return satoshies.toDouble() / 100000000 * this.exchangeRate;
   }
 
+  String formatFiat(double fiatAmount) {
+    return format(fiatToSat(fiatAmount));
```

Comment:
Here we use `fiatToSat` and then use `format` function that converts to fiat again...
Instead we can just extract out the part that formats the fiat to this function and use this function in `format`.

Reply:
[Done](https://github.com/breez/breezmobile/pull/289/commits/d408b9184708ee5526d11090390b652ad04dfae2)